src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -173,7 +173,7 @@ private function computeSubresourceOperations(RouteCollection $routeCollection,
                     '_controller' => self::DEFAULT_ACTION_PATTERN.'get_subresource',
                     '_format' => null,
                     '_api_resource_class' => $subresource,
-                    '_api_subresource_operation_name' => 'get',
+                    '_api_subresource_operation_name' => 'get_'.$operation['property'],
```

Comment:
We need to agree on this because it sets the operation name to override when we want to add a `normalization_context`. 

Maybe `get_subresource_$property`? WDYT ? ping @teohhanhui @dunglas